save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -66,13 +64,34 @@ class DownloadFilesController(
         additionalToolsFileSystemRepository.getFileInfoList(ProjectCoordinates(organizationName, projectName))
     }
 
+    /**
+     * @param organizationName
+     * @param projectName
+     * @param dirName
+     * @param authentication
+     * @return [Mono] with response
+     */
+    @DeleteMapping(path = ["/api/$v1/files/{organizationName}/{projectName}/{dirName}/delete"])
```

Comment:
You probably don't need `/delete` in the path, because it's already specified by http method. Also, since we are already leaking storage implementation in a form of dirName, rename it to `{creationTimestamp}` to at least reduce confusion here?

Reply:
Ok. Fixed